corehq/apps/domain/shortcuts.py

```diff
@@ -25,6 +25,7 @@ def create_user(username, password, is_staff=False, is_superuser=False, is_activ
     else:
         user.password = password
 
+    user.DO_NOT_SAVE_COUCH_USER = True
```

Comment:
Why is this fine to do? Is it because we already explicitly create a couch user as well?

Reply:
Yes. This is called as the first step in creating a new user which means that the Couch user doesn't exist yet. This isn't strictly necessary but it does save a few DB / cache hits and it makes more logical sense to me.

I'll add a comment.